app/scripts/modules/core/src/index.ts

```diff
@@ -17,6 +17,9 @@ export * from './deploymentStrategy';
 export * from './domain';
 
 export * from './entityTag';
+// TODO: try pushing this export back down; for some unknown reason, it causes grief with the library (the export
+// is found by the TS compiler, but not at runtime)
+export * from './entityTag/notifications/EntityNotifications';
```

Comment:
NOTE: This is weird and I don't like it or understand it but I spent almost two hours trying to find a better workaround and I couldn't.